charts/seed-controlplane/charts/alicloud-cloud-controller-manager/templates/alicloud-ccm-deployment.yaml

```diff
@@ -43,6 +43,12 @@ spec:
         garden.sapcloud.io/role: controlplane
         app: kubernetes
         role: cloud-controller-manager
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-private-networks: allowed
+        networking.gardener.cloud/to-metadata: allowed
```

Comment:
No need to access meta service and private network

Reply:
I thought that it was possible to configure the aliyun API endpoint, but I was mistaken https://github.com/kubernetes/cloud-provider-alibaba-cloud/blob/4f03561a5f250cb4f40d15ab3f6fe6e6595dac81/cloud-controller-manager/alicloud.go#L78-L92